blog/_src/posts/2017-02-27-metafunction-apply.md

```diff
@@ -0,0 +1,214 @@
+    Title: PLT Redex: mf-apply
+    Date: 2017-02-27T23:54:20
+    Tags: PLT Redex, package, lang-extension, by Ben Greenman
+
+The `mf-apply` keyword is for checked metafunction application in PLT Redex.
+In other words, `(mf-apply f x)` is just like `(f x)`, but errors if `f` is
+ not a previously-defined metafunction.
+
+Also, consider attending _The Racket School of Semantics and Languages_ this summer:
+[http://summer-school.racket-lang.org/2017/](http://summer-school.racket-lang.org/2017/)
```

Comment:
You might want to write an extra sentence that would let someone get a feeling of whether that is for them: what kind of material is covered, in what form, what is the intended public.